holoviews/plotting/bokeh/callbacks.py

```diff
@@ -53,35 +55,55 @@ def attributes_js(attributes, handles):
 class Callback(object):
     """
     Provides a baseclass to define callbacks, which return data from
-    bokeh models such as the plot ranges or various tools. The callback
+    bokeh model callbacks, events and attribute changes. The callback
     then makes this data available to any streams attached to it.
 
     The definition of a callback consists of a number of components:
 
-    * handles    :  The handles define which plotting handles the
-                    callback will be attached on, e.g. this could be
-                    the x_range, y_range, a plotting tool or any other
-                    bokeh object that allows callbacks.
+    * models     :  Defines which bokeh models the callback will be
+                    attached on referencing the model by its key in
+                    the plots handles, e.g. this could be the x_range,
+                    y_range, plot, a plotting tool or any other
+                    bokeh mode.
 
-    * attributes :  The attributes define which attributes to send
+    * extra_models: Any additional models available in handles which
+                    should be made available in the namespace of the
+                    objects.
```

Comment:
Giving an example of why you might want this would help here. E.g 'For instance, this can be used to check which tools are currently active'.